azure/eventhub_log_forwarder/parent_template.json

```diff
@@ -0,0 +1,101 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "eventHubNamespace": {
+      "type": "string",
+      "defaultValue": "datadog-eventhub-namespace",
+      "metadata": {
+        "description": "Name of EventHub namespace"
+      }
+    },
+    "eventHubName": {
+      "type": "string",
+      "defaultValue": "datadog-eventhub",
+      "metadata": {
+        "description": "Name of Event Hub"
+      }
+    },
+    "functionCode": {
+      "type": "string",
+      "metadata": {
+        "description": "Code for the function to run, saved into index.js"
+      }
+    },
+    "apiKey": {
+      "type": "string",
+      "metadata": {
+        "description": "Datadog API key"
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Specify a location for the resources."
+      }
+    }
+  },
+  "variables": {
+    "eventHubTemplateLink": "https://raw.githubusercontent.com/DataDog/datadog-serverless-functions/master/azure/eventhub_log_forwarder/event_hub.json",
+    "functionAppTemplateLink": "https://raw.githubusercontent.com/DataDog/datadog-serverless-functions/1db05725d9d1fc78a57968e6f8f95053e1d994e0/azure/eventhub_log_forwarder/function_template.json"
```

Comment:
Should be for the master branch

Reply:
Ah yeah, totally forgot to change that. Will update!